chart/templates/replicated-deployment.yaml

```diff
@@ -48,6 +48,12 @@ spec:
       - name: replicated
         secret:
           secretName: {{ include "replicated.secretName" . }}
+      {{- if .Values.privateCAsConfigmap }}
+      - name: kotsadm-private-cas
```

Comment:
let's pick a name without kotsadm in it.  replicated sdk, or something is better?